frontend/src/components/Pages/MissionPage/MissionPage.tsx

```diff
@@ -19,18 +19,48 @@ import { tokens } from '@equinor/eds-tokens'
 import { StyledPage } from 'components/Styles/StyledComponents'
 import { InspectionDialogView, InspectionsViewSection } from '../InspectionReportPage.tsx/InspectionView'
 import { useInspectionsContext } from 'components/Contexts/InpectionsContext'
+import { Typography } from '@equinor/eds-core-react'
 
 const StyledMissionPage = styled(StyledPage)`
     background-color: ${tokens.colors.ui.background__light.hex};
 `
 const TaskAndMapSection = styled.div`
     display: flex;
+    min-height: 60%;
+    padding: 24px;
+    @media (max-width: 600px) {
+        padding: 6px 8px 8px 6px;
+    }
+    flex-direction: column;
+    justify-content: center;
     align-items: flex-start;
+    gap: 8px;
+    align-self: stretch;
+    border-radius: 6px;
+    border: 1px solid ${tokens.colors.ui.background__medium.rgba};
+    background: ${tokens.colors.ui.background__default.rgba};
+`
+
+const StyledTableAndMap = styled.div`
+    display: flex;
     flex-wrap: wrap;
-    gap: 8rem;
-    padding-top: 16px;
-    padding-bottom: 16px;
+    align-items: top;
+    gap: 24px;
+`
+const StyledMissionPageContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
 `
+
+const StyledMissionPageCardsWidth = styled.div`
```

Comment:
```suggestion
const StyledCardsWidth = styled.div`
```
Seems redundant to include the file name in the style name